sounddevice.py

```diff
@@ -570,19 +571,19 @@ def query_devices(device=None, kind=None):
     assert info.structVersion == 2
     name_bytes = _ffi.string(info.name)
     try:
-        # We don't know beforehand if DirectSound and MME device names use
-        # 'utf-8' or 'mbcs' encoding.  Let's try 'utf-8' first, because it more
-        # likely raises an exception on 'mbcs' data than vice versa, see also
-        # https://github.com/spatialaudio/python-sounddevice/issues/72.
-        # All other host APIs use 'utf-8' anyway.
+        # I tried to introduce the locale package to get the
+        # default encoding of the user's system.
+        # And I think there shouldn't be any encoding types other than those two.
+        # But I can't be sure, so I just output bytes.
+        # that way there won't be any more errors caused by decoding failures.
+        # see also 👇
+        # https://github.com/spatialaudio/python-sounddevice/issues/490
         name = name_bytes.decode('utf-8')
     except UnicodeDecodeError:
-        if info.hostApi in (
-                _lib.Pa_HostApiTypeIdToHostApiIndex(_lib.paDirectSound),
-                _lib.Pa_HostApiTypeIdToHostApiIndex(_lib.paMME)):
-            name = name_bytes.decode('mbcs')
-        else:
-            raise
+        try:
+            name = name_bytes.decode(_locale.getpreferredencoding())
+        except UnicodeDecodeError:
+            name = repr(name_bytes)[2:-1]
```

Comment:
What about using this instead?

```suggestion
            name = name_bytes.decode('ascii', errors='backslashreplace')
```

The result should be the same, but this would be more descriptive, I think.

Reply:
But maybe we don't need this fallback anyway.

I think I would prefer to get another error report (just like yours) if it turns out that some system uses yet another unexpected encoding.

Reply:
Maybe we can keep this fallback, but use `print(e)` to output the error so the user can continue to use it or come and submit the error.

Reply:
I'm strongly against using `print()`.